packages/garbo-lib/src/wanderer/freerunquestcompletion.ts

```diff
@@ -0,0 +1,51 @@
+import { appearanceRates, getMonsters, Location } from "kolmafia";
+import { DraggableFight, WandererFactoryOptions, WandererTarget } from "./lib";
+import { get, PeridotOfPeril } from "libram";
+
+export function freeRunQuestCompletionFactory(
```

Comment:
Factories like this are typically named for the target from which they derive their value, not the type of free run used to do them. Free fight is confusing because they're the same thing--a YR is both a target and a type. But, like, other factories are like "guzzlrFactor". Which is to say that this shoudln't be named after freeruns, it should be named after cookbat